src/app/screens/Login/index.tsx

```diff
@@ -1,5 +1,84 @@
 import React from 'react';
+import i18next from 'i18next';
+import { useForm } from 'react-hook-form';
+import { Link } from 'react-router-dom';
+
+import { User } from '~utils/types';
+import { login } from '~services/UserService';
+import { useLazyRequest } from '~app/hooks/useRequest';
+import Loading from '~components/Spinner/components/loading';
+import CustomErrorDisplayer from '~components/CustomErrorDisplayer';
+
+import { PATHS } from '../../../constants/paths';
+import CustomInput from '../../components/CustomInput';
+import WoloxImg from '../Assets/wolox-logo.png';
+
+import { LOGIN_FIELDS } from './constants';
+import styles from './styles.module.scss';
 
 export default function Login() {
-  return <span>Login, ¡coming soon!</span>;
+  const { register, handleSubmit, errors } = useForm<User>({ mode: 'all' });
+  const [state, loading, error, sendRequest] = useLazyRequest({
+    request: login
+  });
+  // eslint-disable-next-line no-console
+  console.log(state?.headers);
+
+  const onSubmit = handleSubmit(data => {
+    sendRequest(data);
+  });
+  return (
+    <div className="column center">
+      <form className={`column ${styles.loginForm}`} onSubmit={onSubmit}>
+        <img src={WoloxImg} alt={i18next.t('Login:logoAlt') as string} className={styles.logo} />
+        <CustomInput
+          name={LOGIN_FIELDS.email}
+          inputRef={register({
+            required: { value: true, message: i18next.t('Login:required') },
+            pattern: {
+              value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
+              message: i18next.t('Login:invalidEmail')
+            }
+          })}
+          error={errors.email?.message}
+          label={i18next.t('Login:email') as string}
+          labelClassName={styles.customLabel}
+          type="email"
+        />
+        <CustomInput
+          name={LOGIN_FIELDS.password}
+          inputRef={register({
+            required: { value: true, message: i18next.t('Login:required') }
+          })}
+          error={errors.password?.message}
+          label={i18next.t('Login:password') as string}
+          labelClassName={styles.customLabel}
+          type="password"
+        />
+        <div className={`column ${styles.loginButtonContainer}`}>
+          <button
+            className={`${styles.customButton} ${styles.loginButton}
+            ${loading ? styles.disabled : ''}
+            ${Object.keys(errors).length > 0 ? styles.disabled : ''}
```

Comment:
Try a package named clsx it's nice for conditional classNames, use it only when you have conditionals